addons/mass_mailing/static/src/js/mass_mailing_wysiwyg.js

```diff
@@ -70,6 +91,34 @@ export class MassMailingWysiwyg extends Wysiwyg {
         this.mainToolbarEl.classList.toggle('d-none', !fold);
     }
 
+    /**
+     * @override
```

Comment:
If we're going to add a docstring, might as well briefly explain what we're doing here and why.